src/utils/format.ts

```diff
@@ -14,11 +13,9 @@ function _getLocaleSymbols(): { thousands: string; decimals: string } {
   }
 }
 const { thousands: THOUSANDS_SYMBOL, decimals: DECIMALS_SYMBOL } = _getLocaleSymbols()
-
```

Comment:
What do you suddenly have against new lines between functions?

Reply:
no idea why I deleted that lines. I assume it was my Visual Studio